modules/msk/mutect1/tests/nextflow.config

```diff
@@ -0,0 +1,12 @@
+params {
+    enable_conda = false
```

Comment:
we can remove this for now.

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc